lib/services.js

```diff
@@ -48,12 +48,14 @@ module.exports = function generateServices(app, options) {
     options = {
       ngModuleName: arguments[1],
       apiUrl: arguments[2],
+      modelsToIgnore: arguments[3] ? arguments[3] : []
```

Comment:
This branch is for the legacy API that should not be extended by new features. Please remove this line.